common/src/main/scala/org/apache/celeborn/common/CelebornConf.scala

```diff
@@ -2110,6 +2111,14 @@ object CelebornConf extends Logging {
       .timeConf(TimeUnit.SECONDS)
       .createWithDefaultString("10min")
 
+  val COMPUTE_RESOURCE_CONSUMPTION_INTERVAL: ConfigEntry[Long] =
+    buildConf("celeborn.compute.userResourceConsumption.interval")
```

Comment:
Let's rename it to "celeborn.metrics.userResourceConsumption.update.interval" which follows existing "celeborn.shuffle.estimatedPartitionSize.update.interval"

Reply:
DOne